vpr/src/route/router_lookahead_map_utils.h

```diff
@@ -26,7 +26,7 @@
 #include "rr_graph_view.h"
 
 /* we will profile delay/congestion using this many tracks for each wire type */
-#define MAX_TRACK_OFFSET 16
+static constexpr int MAX_TRACK_OFFSET = 16;
```

Comment:
Add comment: larger values increase the time to compute the lookahead, but may give more accurate lookahead estimates during routing.